modules/social_features/social_group/config/install/views.view.newest_groups.yml

```diff
@@ -32,7 +32,7 @@ display:
         type: views_query
         options:
           disable_sql_rewrite: false
-          distinct: false
+          distinct: true
```

Comment:
This comes at quite the performance cost in the query execution time, I would love to see if we can find an alternative method to fix this issue if we can.

Reply:
Hi @ronaldtebrake 
Thank you!
I change approach and use GROUP BY instead of DISTINCT
Please, check